common/scala/src/main/scala/whisk/core/containerpool/HttpUtils.scala

```diff
@@ -76,11 +83,12 @@ protected[core] class HttpUtils(hostname: String, timeout: FiniteDuration, maxRe
     request.addHeader(HttpHeaders.ACCEPT, "application/json")
     request.setEntity(entity)
 
-    execute(request, timeout.toMillis.toInt, retry)
+    execute(request, timeout.toMillis.toInt, maxConcurrent, retry)
   }
 
   private def execute(request: HttpRequestBase,
                       timeoutMsec: Integer,
+                      maxConcurrent: Int,
```

Comment:
how is `maxConcurrent` used in `execute`?

Reply:
hmm right. must have been left over from some previous iteration. removed it